l10n_it_intrastat_statement/models/intrastat_statement.py

```diff
@@ -775,35 +775,42 @@ def compute_statement(self):
             inv_type += ["in_invoice", "in_refund"]
         domain.append(("move_type", "in", inv_type))
 
-        statement_data = dict()
+        statement_data = {}
+        section_field_reverse = {}
+        for section_type in ["purchase", "sale"]:
+            for section_number in range(1, 5):
+                statement_section_field = self.get_section_field_name(
+                    section_type, section_number
+                )
+                statement_data[statement_section_field] = list()
+                section_model = self.get_section_model(
+                    section_type=section_type, section_number=section_number
```

Comment:
```suggestion
                    section_type, section_number
```
Questi non sono `kwargs` ma `args`, quindi si possono assegnare semplicemente in base alla loro posizione.

Reply:
ok! Lo cambio